test/rules/structured-imports/baseCase.ts.lint

```diff
@@ -0,0 +1,3 @@
+import Foo from './foo';
```

Comment:
`baseCase` is too generic.  I should have a _rough_ idea of what a test is checking from the filename.